help-beta/src/components/NavigationSteps.astro

```diff
@@ -138,6 +151,135 @@ const setting_link_mapping: {
+      },
+      "organization-settings": {
+        label: `${RawZulipIconBuilding} Organization settings`,
+        relative_link: "/#organization/organization-profile",
+      },
+      "group-settings": {
+        label: `${RawZulipIconUserGroupCog} Group settings`,
+        relative_link: "/#groups/your",
+      },
+      stats: {
+        label: `${RawZulipIconBarChart} Usage statistics`,
+        relative_link: "/stats",
+      },
+      integrations: {
+        label: `${RawZulipIconGitPullRequest} Integrations`,
+        relative_link: "/integrations/",
+      },
+      "about-zulip": {
+        label: "About Zulip",
+        relative_link: "/#about-zulip",
+      },
+    },
+    template: default_template_for_relative_links,
+    is_link_relative: () => SHOW_RELATIVE_LINKS,
+  },
+  "gear-billing": {
+    data: {
+      plans: {
+        label: `${RawZulipIconRocket} Plans and pricing`,
+        relative_link: "/plans/",
+      },
+      billing: {
+        label: `${RawZulipIconCreditCard} Billing`,
+        relative_link: "/billing/",
+      },
+    },
+    template: default_template_for_relative_links,
+    is_link_relative: () => SHOW_RELATIVE_LINKS && SHOW_BILLING_HELP_LINKS,
+  },
+  help: {
+    data: {
+      "keyboard-shortcuts": {
+        label: `${RawZulipIconKeyboard} Keyboard shortcuts`,
+        relative_link: "/#keyboard-shortcuts",
+      },
+      "message-formatting": {
+        label: `${RawZulipIconEdit} Message formatting`,
+        relative_link: "/#message-formatting",
+      },
+      "search-filters": {
+        label: `${RawZulipIconManageSearch} Search filters`,
+        relative_link: "/#search-operators",
+      },
+      "about-zulip": {
+        label: `${RawZulipIconInfo} About Zulip`,
+        relative_link: "/#about-zulip",
+      },
+    },
+    template: default_template_for_relative_links,
+    is_link_relative: () => SHOW_RELATIVE_LINKS,
+  },
+  channel: {
+    data: {
+      all: {
+        label: "All",
+        relative_link: "/#channels/all",
+      },
+      "not-subscribed": {
+        label: "Not subscribed",
+        relative_link: "/#channels/notsubscribed",
+      },
+    },
+    template: `
+<ol>
+  <li>Click on the <strong>gear</strong> (${RawZulipIconGear}) icon in the upper right corner of the web or desktop app.</li>
+  <li>Select ${RawZulipIconHash} <strong>Channel settings</strong>.</li>
+  <li>Click {item} in the upper left.</li>
+</ol>
+`,
+    is_link_relative: () => SHOW_RELATIVE_LINKS,
+  },
+  group: {
+    data: {
+      all: {
+        label: "All groups",
+        relative_link: "/#groups/all",
+      },
+    },
+    template: `
+<ol>
+  <li>Click on the <strong>gear</strong> (${RawZulipIconGear}) icon in the upper right corner of the web or desktop app.</li>
+  <li>Select ${RawZulipIconUserGroupCog} <strong>Group settings</strong>.</li>
+  <li>Click {item} in the upper left.</li>
+</ol>
+`,
+    is_link_relative: () => SHOW_RELATIVE_LINKS,
+  },
+};
```

Comment:
This seems readable enough that we should be content maintaining it.

I didn't try to carefully check the conversion matches the original behavior; my theory is the diffing tool is the way to review for mistakes there.